### DISCUSSION TITLE
Flutter doesn't recommend navigation using named routes

### DISCUSSION THREAD

**Discussion by jeremyhalin** [nylo-core/nylo#32] [Ideas]
Hi,

I've seen in the Flutter's documentation that[ they do not recommend named routes](https://docs.flutter.dev/development/ui/navigation#limitations).

I don't see a package in nylo framework that handles routing. Does it mean that nylo is still using named routes?

**Comment by agordn52** [comment]
Hi,

I'll confirm back here with my findings.
